### PR TITLE
Fixed tree bug

### PR DESCRIPTION
**Pull request by svouse**
Changed a line in makeTreeListener

---
This change is [Reviewable](https://reviewable.io/reviews/google/blockly-devtools/19)

### CODE REVIEW

**Comment by rachel-fenichel** [comment]
Looks good.  Please resolve the issue with CLAs (follow github's instructions) and then it'll be good to merge.

File: src/block_library_controller.js [hunk 1]
```diff
@@ -370,13 +370,14 @@ BlockLibraryController.prototype.buildTree = function() {
...
+  var lib = this;
   $('#navigationTree').on('changed.jstree', function (e, data) {
     // collect data of all selected blocks
     var i, j, r = [];
     for (i = 0, j = data.selected.length; i < j; i++) {
       r.push(data.instance.get_node(data.selected[i]).text);
     }
     // load the blocks
-    this.openBlock(r.join(', '));
+    lib.openBlock(r.join(', '));
```
Comment (AnmAtAnm) [inline src/block_library_controller.js:381]: Another alternative is using `.bind(this)`:

```
on('event', function(e) {
  // ...
  this.whatever(..)
}.bind(this));
```

**Comment by AnmAtAnm** [comment]
Regarding the CLA, you may need to make sure your commits are using the same email as your github account. See https://help.github.com/articles/setting-your-email-in-git/

Then `commit --amend` to update the  and `push -f` (force)

**Comment by rachel-fenichel** [comment]
What's going on with these new commits?

**Comment by svouse** [comment]
My bad; had been committing to wrong branch

File: index.html [hunk 2]
```diff
@@ -770,7 +770,6 @@
   </div>
 </div>
   <!-- Navigation Tree -->
-  <div id="navigationTree"></div>
-
+   <div id= "navigationTree"></div>
```
Comment (AnmAtAnm) [inline index.html]: Added an extra space on the indent

File: src/app_controller.js [hunk 3]
```diff
@@ -530,10 +531,10 @@ AppController.prototype.assignBlockFactoryClickHandlers = function() {
       }
 
       this.createBlocklyInitPopup(false);
-      self.blockLibraryController.setNoneSelected();
+    self.blockLibraryController.setNoneSelected();
 
       // Close the Block Library Dropdown.
-      self.closeModal();
+     self.closeModal();
```
Comment (AnmAtAnm) [inline src/app_controller.js]: Correct both indents above

File: src/block_library_controller.js [hunk 4]
```diff
@@ -370,13 +365,14 @@ BlockLibraryController.prototype.buildTree = function() {
 * Listen for block selected in tree
 */
 BlockLibraryController.prototype.makeTreeListener = function() {
+  var lib= this;
```
Comment (AnmAtAnm) [inline src/block_library_controller.js]: space after lib. `self` is the more common name in this pattern.

File: src/workspacefactory/wfactory_controller.js [hunk 5]
```diff
@@ -320,7 +320,7 @@ WorkspaceFactoryController.prototype.clearAndLoadElement = function(id) {
 WorkspaceFactoryController.prototype.exportXmlFile = function(exportMode) {
   // Get file name.
   if (exportMode == WorkspaceFactoryController.MODE_TOOLBOX) {
-    var fileName = prompt('File Name for toolbox XML:', 'toolbox.xml');
+    var fileName = prompt('File Name for toolbox JavaScript:', 'toolbox.js');
```
Comment (AnmAtAnm) [inline src/workspacefactory/wfactory_controller.js]: Add a TODO about supporting XML for other platforms

File: src/block_library_controller.js (hunk 1 above)
Comment (AnmAtAnm) [reply src/block_library_controller.js:381]: Or better yet, just use [ES6 lambdas](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions):

```
on('event', (e, data) => {
  // ...
  this.whatever(..)
});
```

**Review by AnmAtAnm — APPROVED** [review]

**Comment by AnmAtAnm** [comment]
LGTM

**Pull request merged**